src/Process/Pool/Strategy/Server.php

```diff
@@ -0,0 +1,69 @@
+<?php
+declare(strict_types=1);
+
+namespace Neighborhoods\Kojo\Process\Pool\Strategy;
+
+use Neighborhoods\Kojo\Process\Pool\StrategyAbstract;
+use Neighborhoods\Kojo\Process\Root;
+use Neighborhoods\Kojo\ProcessInterface;
+use Neighborhoods\Kojo\Process\Pool\StrategyInterface;
+
+class Server extends StrategyAbstract
+{
+    protected $_pausedListenerProcesses = [];
+
+    public function childProcessExited(ProcessInterface $process): StrategyInterface
+    {
+        if ($process instanceof Root) {
+            $this->_getProcessPool()->freeChildProcess($process->getProcessId());
+            $rootProcess = $this->_getProcessCollection()->getProcessPrototypeClone($process->getTypeCode());
+            $this->_getProcessPool()->addChildProcess($rootProcess);
+        } else {
+            $className = get_class($process);
+            throw new \UnexpectedValueException("Unexpected process class[$className].");
```

Comment:
This is because `Worker` processes are expected to be children of the `Root` process, correct?

Reply:
yes, correct.